WireAPI/Tests/WireAPITests/APIs/ConversationsAPI/ConversationsAPITests.swift

```diff
@@ -210,4 +214,345 @@ final class ConversationsAPITests: XCTestCase {
             XCTFail("expected error 'FailureResponse'")
         }
     }
+
+    // MARK: getConversations
+
+    func testGetConversations_givenV0_thenVerifyRequests() async throws {
```

Comment:
I only see snapshots for V0, V2, V3, and V5. I think though we should have snapshots for all versions, since they all have different paths.

Perhaps we should have a `verifyRequestForAllVersions` test like we do in other API tests?

Reply:
right, done in 1a5ca4a42ba9d1b4d63d8ca5b2b49378531a9797 and ecf8b97cb348016943f1eaed9db93c5db66e12de